content/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -39,6 +40,20 @@ This document is the record of all the changes in the **Customer Chat API**, Web
 - Method **List Chats** ([Web](/messaging/customer-chat-api/v3.2/#list-chats) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#list-chats)) is paginated with `page_id` now.
 - The [Login](/messaging/customer-chat-api/v3.2/rtm-reference/#login) method was extended with the application info.
 - The **Get Chat Threads** method evolved into **Get Chat** ([Web](/messaging/customer-chat-api/v3.2/#get-chat) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#get-chat)). It now allows to retrieve a chat containing a particular thread. It also changed its shape; it no longer contains the `order` field and the `chat` key.
+- There's a new field `queue` in the [**Thread**](/messaging/customer-chat-api/v3.2/#threads) data structure.
+- There's a new field `requester_id` in [`incoming_chat`](/messaging/customer-chat-api/v3.2/rtm-reference/#incoming_chat).
+- [`chat_transferred`](/messaging/customer-chat-api/v3.2/rtm-reference/#chat_transferred) push is no longer tied directly to **Transfer Chat** ([Web](/messaging/agent-chat-api/v3.2/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#transfer-chat)), can now be generated when the transfer is implicit e.g. the agents are reassigned due to their inactivity.
```

Comment:
```suggestion
- The [`chat_transferred`](/messaging/customer-chat-api/v3.2/rtm-reference/#chat_transferred) push is no longer tied directly to **Transfer Chat** ([Web](/messaging/agent-chat-api/v3.2/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#transfer-chat)). Now, it can be generated when the transfer is implicit, for example, agents are reassigned due to their inactivity.
```